examples/yoti_example_django/requirements.in

```diff
@@ -1,7 +1,17 @@
-cryptography>=2.3
 Django==2.2.4
 django-sslserver>=0.2.0
 python-dotenv>=0.7.1
 requests>=2.20.0
 urllib3>=1.24.2
-yoti>=2.9.0
+asn1==2.2.0
```

Comment:
Can we change it so that we don't pin the SDK dependencies in the example projects?  Can we get the docker file to install the SDK's dependencies before changing work dir, then install the example project's dependencies?

Reply:
I think that might be better if we can do that. Otherwise we're going to have to update the dependencies here whenever we update them in the SDK too

Reply:
I was worried about adding the extra step to running it without Docker.

Reply:
So to confirm, clients won't need to run an extra step now when running without Docker unless they want a non-default version of the Yoti package?